hooks/pre_prompt.py

```diff
@@ -0,0 +1,12 @@
+import warnings
+
+from ccds.version import __version__ as version
+
+if __name__ == "__main__":
+    if version <= "2.0.0":
+        warnings.warn(
+            "It looks like you're using a version of CCDS that "
+            "always uses the latest template. For more stable behavior, "
+            "update to a newer version of CCDS.",
+            DeprecationWarning,
+        )
```

Comment:
[nit] my brain found it easier to reason about < 2.0.1 than <= 2.0.0 ("if the version is less than 2.0.1, show the deprecation warning")

Then tweaked the wording a bit -- still correct?

```suggestion
    if version < "2.0.1":
        warnings.warn(
            "It looks like you're using a version of CCDS that "
            "defaults to using the latest template. For more stable behavior, "
            "update to CCDS version 2.0.1 or later.",
            DeprecationWarning,
        )
```

Reply:
Would be good to tell people how to upgrade (e.g., `pip install -U cookiecutter-data-science`)

Reply:
Good call -- updated suggestion.